systemd/ceph-mds@.service

```diff
@@ -11,14 +11,15 @@ EnvironmentFile=-/etc/sysconfig/ceph
 Environment=CLUSTER=ceph
 ExecStart=/usr/bin/ceph-mds -f --cluster ${CLUSTER} --id %i --setuser ceph --setgroup ceph
 ExecReload=/bin/kill -HUP $MAINPID
-PrivateDevices=yes
+PrivateDevices=no
```

Comment:
Regarding ulimit –l, due to a bug in RH/Centos we were unable to set a value to Memlock param. only Infinity value is accepted. Hoping this will be fixed.

https://access.redhat.com/solutions/2854131 

 

As for "PrivateDevice”. This is a MUST for RDMA.

Since systemd file is executed before any ceph code what I can suggest is adding a script to parse ceph.conf and look if RDMA configuration is needed. 

Sounds OK?